tools/msstatstmt/msstatstmt.xml

```diff
@@ -36,20 +44,34 @@ library('MSstatsTMT', warn.conflicts = F, quietly = T, verbose = F)
                                     rmProtein_with1Feature = $input.input_options.rmProtein_with1Feature,
                                     summaryforMultipleRows = $input.input_options.summaryforMultipleRows)
 
-    quant <- proteinSummarization(input,
-                                  method = "$proteinSummarization.method",
-                                  global_norm = $proteinSummarization.global_norm,
-                                  reference_norm = $proteinSummarization.reference_norm,
-                                  remove_norm_channel = $proteinSummarization.remove_norm_channel,
-                                  remove_empty_channel = $proteinSummarization.remove_empty_channel,
-                                  MBimpute = $proteinSummarization.MBimpute,
-                                  #if $proteinSummarization.maxQuantileforCensored.select == 'false'
-                                  maxQuantileforCensored = NULL)
-                                  #else
-                                  maxQuantileforCensored = $proteinSummarization.maxQuantileforCensored.value)
-                                  #end if
+#elif $input.input_src == 'OpenMS'
+    #if $input.oms_input.is_of_type('csv')
```

Comment:
If you like I think you can drop the support of CSV. If a user realy needs to use CSV, galaxy can/should convert CSV to tabular before you use the tool.